### PR TITLE
Implemented: app version component from dxp-components(#176)

### LINKED ISSUE #176: Integrate DXP Component: App Version Info on the Settings Page
## What is the motivation for adding/enhancing this feature?
IntegratE of the DXP component "App Version Info" into the Settings page of the application. This addition will provide users with essential information about the version of the application they are currently using. 

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by Ritika-Patel08**
### Related Issues

Related issue  #176

### Short Description and Why It's Useful

Used app version component from the dxp-components in the settings page.

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -29,7 +29,8 @@ import i18n from './i18n'
 import store from './store'
 import { dxpComponents } from '@hotwax/dxp-components'
 import { login, logout, loader } from './user-utils';
-import { getConfig, getProductIdentificationPref, initialise, setProductIdentificationPref } from '@/adapter'
+import { getConfig, initialise, getProductIdentificationPref, setProductIdentificationPref } from '@/adapter'
```
Comment (amansinghbais) [inline src/main.ts:32]: Remove unwanted changes.

File: src/views/Settings.vue [hunk 2]
```diff
@@ -177,9 +168,6 @@ export default defineComponent({
     goToLaunchpad() {
       window.location.href = `${process.env.VUE_APP_LOGIN_URL}`
     },
-    getDateTime(time: any) {
-      return DateTime.fromMillis(time).toLocaleString(DateTime.DATETIME_MED);
```
Comment (amansinghbais) [inline src/views/Settings.vue:181]: Dont import DateTime as not used.

File: src/main.ts (hunk 1 above)
Comment (Ritika-Patel08) [reply src/main.ts:32]: done

File: src/views/Settings.vue (hunk 2 above)
Comment (Ritika-Patel08) [reply src/views/Settings.vue:181]: Done, You can verify.

File: src/main.ts (hunk 1 above)
Comment (amansinghbais) [reply src/main.ts:32]: Not resolved.

Comment (Ritika-Patel08) [reply src/main.ts]: Now Its resolved.
You can verify.

**Review by amansinghbais — APPROVED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**